src/views/Nominate.vue

```diff
@@ -1,59 +1,82 @@
 <template>
-    <div class="nominate">
-        <div class="nominate__section">
-            <div class="nominate__content-wrapper">
-                <h1 class="nominate__title">
-                    Nominate a <span class="nominate__highlight">developer</span>
-                </h1>
-                <p class="nominate__body-text">
-                    Help us build the directory of Developers thriving in the field of programming.
-                    If you know someone who deserves to be on the list, nominate them by clicking the Nominate button and filling the form.
-                    And hey, it's alright to nominate yourself. 😎
-                </p>
-                <a href="#" rel="noreferrer" class="btn primary">Nominate</a>
-            </div>
-        </div>
+  <div class="nominate">
+    <div class="nominate__section">
+      <div class="nominate__content-wrapper">
+        <h1 class="nominate__title">
+          Nominate a <span class="nominate__highlight">developer</span>
+        </h1>
+        <p class="nominate__body-text">
+          Help us build the directory of Developers thriving in the field of
+          programming. If you know someone who deserves to be on the list,
+          nominate them by clicking the Nominate button and filling the form.
+          And hey, it's alright to nominate yourself. 😎
+        </p>
+
+        <button class="btn primary" @click="isOpen != isOpen">Nominate</button>
+
+        <template v-if="isOpen">
+            <NominateForm />
+        </template>
```

Comment:
To be honest, I think this should be visible always.
```suggestion
        <NominateForm />

```

Reply:
In this case, the Nominate button would be useless ... or not?

Reply:
Yep